AnalysisDataFormats/EWK/src/classes_def.xml

```diff
@@ -12,14 +12,10 @@
    <class name="edm::helpers::Key<edm::RefProd<std::vector<reco::WMuNuCandidatePtr> > >"/>
 
    <class name="edm::reftobase::BaseHolder<reco::WMuNuCandidatePtr>"/>
-   <class name="edm::RefToBase<reco::WMuNuCandidatePtr>"/>
    <class name="edm::reftobase::IndirectHolder<reco::WMuNuCandidatePtr>" />
    <class name="edm::RefToBaseProd<reco::WMuNuCandidatePtr>" />
-   <class name="edm::RefToBaseVector<reco::WMuNuCandidatePtr>" />
-   <class name="edm::Wrapper<edm::RefToBaseVector<reco::WMuNuCandidatePtr> >" />
    <class name="edm::reftobase::BaseVectorHolder<reco::WMuNuCandidatePtr>" />
```

Comment:
By the way, the dictionaries for all these auxiliary template instantiations (with `reftobase` as part of their name) should be removed as well.

Was the removal of the `WMuNuCandidate` and `WMuNuCandidatePtr` altogether (as they do not seem to be used in CMSSW) considered?

Reply:
The SMP conveners came back to me very quickly, and they say that it seems nobody is using this object anymore, so it can be removed!